portfolio/src/main/webapp/script.js

```diff
@@ -36,4 +35,69 @@ function createCommentElement(userName, comment) {
   divElement.innerText = userName + ': ';
   divElement.appendChild(pElement);
   return divElement;
-}
\ No newline at end of file
+}
+
+function getLoggedInStatus() {
+  fetch('/data').then(response => response.json()).then((json) => {
+    const loginButton = document.getElementById('login-logout');
+    const loggedIn = json.loggedInStatus;
+    if (loggedIn) {
+      // add 'logged in as __, [logout]'
+      // add form
+      createCommentForm();
+    }else {
+      // login to comment
+      const loginButton = document.createElement('button');
+      loginButton.innerText = 'LOGIN TO ADD A COMMENT';
+      loginButton.setAttribute('class', 'btn btn-outline-dark custom-btn title-font');
+      addCommentElement.appendChild(loginButton);
+    }
+  });
+}
+
+function createCommentForm() {
```

Comment:
You could do display:none or other approaches to hide for the form element. Then this function can just revert that
https://allyjs.io/tutorials/hiding-elements.html

Reply:
Done: 2dfab35 
This is useful. Thank you.